Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs

```diff
@@ -205,5 +206,40 @@ public JsonResult UpdateParticipantAnswer([DataSourceRequest]DataSourceRequest r
 
             return this.GridOperation(request, model);
         }
+
+        public ActionResult NormalizeParticipants()
+        {
+            var problems = this.Data.Problems.All().ToList();
+            
+            foreach (var problem in problems)
+            {
+                var data = new OjsData();
```

Comment:
Remove the creation of a new OjsData, it is already passed to the constructor of the controller (you used this.Data 5 lines above)